src/fixtureindex.ts

```diff
@@ -10,6 +10,8 @@ export interface IndexItem {
   path?: string
   sha?: string
   url?: string
+  make?: string
+  model?: string
```

Comment:
Here this can be `undefined` b/c this will only be available if we perform a `deepSync`.